config/system-stats-monitor.json

```diff
@@ -15,7 +15,19 @@
       },
       "cpu/load_15m": {
 				"displayName": "cpu/load_15m"
-			}
+      },
+      "system/processes_total": {
+        "displayName": "system/processes_total"
+      },
+      "system/procs_running": {
+        "displayName": "system/procs_running"
```

Comment:
Can we make the display name the same:
system/processes_running?

Reply:
This is consistent with the `/proc/stat` usage:

```
processes 2915
procs_running 1
procs_blocked 0
```

I would also like to not change it since our internal config will need to be changed to be consistent and I want to avoid making such changes.